pkg/cmd/render/manifests.go

```diff
@@ -45,7 +45,7 @@ func (o *RenderManifestsOptions) Run() error {
 	if len(params.ExternalOauthDNSName) == 0 {
 		params.ExternalOauthDNSName = params.ExternalAPIDNSName
 	}
-	err = render.RenderClusterManifests(params, o.PullSecretFile, o.OutputDir, externalOauth, o.IncludeRegistry)
+	err = render.RenderClusterManifests(params, o.PullSecretFile, o.OutputDir, externalOauth, o.IncludeRegistry, params.KonnectivityEnabled)
```

Comment:
This data is already passed in via `params`.  There is no need to add this parameter.